scalableminds/webknossos
# Add upgrade motivation banner

### Steps to test:
- mock an outdated WK version, e.g. by mocking getBuildInfo(), see comment in maintenance_banner.tsx
- see banner, make sure it looks good (e.g. across different themes)
- test link
- click it away 
- it shouldn't be shown again, even if you log out / close the browser. it should only be shown again after another 3 days
- it should be shown again in a private browser tab though.

### TODOs:
- [x] fix styling of button, find out how to do theming well
- [x] fix banner when browser window is narrow 
- [x] fix upcoming maintenance banner when user is logged out


### Issues:
- fixes #7745 

------
(Please delete unneeded items, merge only when none are left open)
